Packs/FeedProofpoint/Integrations/FeedProofpoint/FeedProofpoint.py

```diff
@@ -109,15 +110,34 @@ def _build_iterator(
 
     @staticmethod
     def _process_item(item: dict, tags: list, tlp_color: Optional[str] = None) -> dict:
+        first_seen_raw = item.get("first_seen","")
+        last_seen_raw = item.get("last_seen","")
+
+        if first_seen_raw:
+            try:
+                first_seen = dateparser.parse(first_seen_raw).isoformat()
+            except AttributeError:
+                first_seen = ""
+        else:
+            first_seen = ""
+
+        if last_seen_raw:
+            try:
+                last_seen = dateparser.parse(last_seen_raw).isoformat()
+            except AttributeError:
+                last_seen = ""
+        else:
+            last_seen = ""
```

Comment:
Consider creating a function that takes in a value of type string, and tries to run **dateparser.parse(value).isoformat()** on it and returns, and if there is an **AttributeError** exception, then return an empty string, then use that function on both **item.get("first_seen","")** and **item.get("last_seen","")**, resulting in less code duplication.